docs/management/maintenance-windows/maintenance-windows.asciidoc

```diff
@@ -0,0 +1,49 @@
+[[maintenance-windows]]
+== Maintenance windows
+:description: Maintenance windows enable you to suppress rule notifications.
+:tags-products: [kibana, alerting] 
+:tags-content-type: [overview] 
+:tags-user-goals: [manage]
+
+preview::[]
+
+You can schedule single or recurring maintenance windows to temporarily reduce rule notifications.
+For example, a maintenance window prevents false alarms during planned outages.
+
+Maintenance windows affect all rules in all {kib} apps and spaces.
+Alerts continue to be generated, however notifications are suppressed as follows:
+
+- When an alert occurs during a maintenance window, there are no notifications.
+When the alert recovers, there are no notifications--even if the recovery occurs after the maintenance window ends.
+- When an alert occurs before a maintenance window and recovers during or after the maintenance window, notifications are sent as usual.
+
+[discrete]
+[[setup-maintenance-windows]]
+== Configure access to maintenance windows
+
+To use maintenance windows, you must have the appropriate {subscriptions}[subscription] and {kib} feature privileges.
+
+- To have full access to maintenance windows, you must have `All` privileges for the *Management > Maintenance Windows* feature.
+- To have view-only access to maintenance windows, you must have `Read` privileges for the **Management > Maintenance Windows* feature.
+
+For more details, refer to <<kibana-privileges>>.
+
+[discrete]
+[[manage-maintenance-windows]]
+== Create and manange maintenance windows
+
+In *Management > {stack-manage-app} > Maintenance Windows*, you can create, edit, and archive maintenance windows.
+
+When you create a maintenance window, you must provide a name and a schedule.
+You can optionally configure it to repeat daily, monthly, yearly, or on a custom interval.
+
+[role="screenshot"]
+image::images/create-maintenance-window.png[The Create Maintenance Window user interface in {kib}]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+A maintenance window can have any one of the following statuses:
+
+- `Upcoming`: It will run at the scheduled date and time.
+- `Running`: It is running.
+- `Finished`: It ended and does not have a repeat schedule.
+- `Archived`: It ended and is queued for deletion.
```

Comment:
So we don't actually have an auto deletion mechanism at the moment, it's something we are looking to implement for a future release (not planned for 8.9), so right now archived maintenance windows are exactly the same as finished maintenance windows

Reply:
Thanks!  I've changed that description in 64a0b00e70285fca5b15d18362a222ee5d11bd7a